Misc/NEWS.d/next/Documentation/2019-07-16-14-48-12.bpo-37256.qJTrBb.rst

```diff
@@ -0,0 +1 @@
+change wording in Request class
```

Comment:
Specified location of change and added RST class and module tags ([devguide](https://devguide.python.org/documenting/#inline-markup)):
```suggestion
Fix wording of arguments for :class:`Request` in :mod:`urllib.request`
```